src/components/Navbar.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import { Button, CustomLink } from "@/atoms";
+import { useRouter } from "next/navigation";
+import Image from "next/image";
+
+export const Navbar = () => {
+  const router = useRouter();
+
+  const navigateOnClick = (path: string) => router.push(path);
+
```

Comment:
I think we will eventually want to pass this navigation function down as a prop but for now leaving it here.